frontend/src/modules/app/ToolBar.tsx

```diff
@@ -161,16 +189,33 @@ export const ToolBar = ({
     string,
     Range
   >;
+  /**
+   * State to manage toggling ventilationState
```

Comment:
What is this `isVentilatorOn` state used for? What is its relationship with the `ventilating` flag in `ParametersRequest` and `Parameters`?

Reply:
We were using `isVentilatorOn` as a local state to toggle between Start/Pause to change the local label to 'Start' or 'Pause' & redirect immediately to dashboard page.

But after implementing feature of redirecting to dashboard page only when backend is `ventilating`,  We are not using isVentilatorOn anymore & can be removed